### ISSUE TITLE
Subimages are incorrectly named.

### ISSUE THREAD

**Issue by acviana** [STScI-Citizen-Science/MTPipeline#70]
The png subimage are not correctly named.

**Comment by acviana** [comment]
Cory recently sent us an image that indicated that the images were not being labeled correctly:

![image_sets](https://f.cloud.github.com/assets/1630700/896832/c0fa568c-fae6-11e2-8f1d-6f0c4c6cc62a.png)

This suggests that the subimages are labeled:

| 3 | 6 | 9 | 12 | 
| 2 | 5 | 8 | 11 | 
| 1 | 4 | 7 | 10 | 

Here is how the subimages should be labeled:

| 1 | 4 | 7 | 10 |
| 2 | 5 | 8 | 11 |
| 3 | 6 | 9 | 12 |

![mtpipelinegrid](https://f.cloud.github.com/assets/1630700/896801/07829520-fae6-11e2-8f16-a6fadcd7a24e.JPG)

**Comment by acviana** [comment]
@prathamjoshi made a test image we can use for this:

![gridpng](https://f.cloud.github.com/assets/1630700/896869/78429dfe-fae7-11e2-9014-f61a88f7e167.png)

**Comment by acviana** [comment]
Just using some basic Python I was able to reproduce what I think could be the problem:

``` python
import numpy
from PIL import Image
data = numpy.asarray(Image.open('/Users/viana/Downloads/gridpng.png'))
from matplotlib import pylab
pylab.imshow(data)
```

Which plots the image upside down:

![pnggridflipped](https://f.cloud.github.com/assets/1630700/896893/c61dfed8-fae7-11e2-862b-86d362f6baff.png)

The reason we don't see this in our outputs is because the image is flipped before it's written out:

``` python
def make_png(data, filename):
    '''
    Use the Python image library (PIL) to write out the png file. The 
    data is flipped in the "up-down" direction and converted to 8bit
    encoding before writing.
    '''
    assert isinstance(data, N.ndarray), 'array must be numpy array'
    data = N.flipud(data)
    data = N.uint8(data)
    image = Image.new('L', (data.shape[1], data.shape[0]))
    image.putdata(data.ravel())
    image.save(filename)
```

This works fine when we are dealing with the entire image, but since we work with the upside down image in the code this means that each subimage is coming from the upside down image not the regular image. Flipping it restore the correct orientation but not where it was cut from.

**Comment by acviana** [comment]
OK, I think I've fixed this. Here is one of the subimage of region 3 now showing the correct information:

![u2mi0a01t_cr_c0m_wide_single_sci_linear_3](https://f.cloud.github.com/assets/1630700/906577/856e3850-fcb6-11e2-9858-ec78a0fe8f8a.png)

**Comment by acviana** [comment]
Resolved in 4d691c33c82508b38c54503df098cb9a592e9b60
